ex04.ex

```diff
@@ -75,7 +79,18 @@ defmodule Ex04 do
   return value will be the thing you have to manipulate.
   """
 
-  def even_odd . . . "your code"
+  #def even_odd . . . "your code"
```

Comment:
## -3

The way you add the accumulators in the first function head  is elegant.

I think you could go a little further in splitting the code into functions, and at the same time eliminate some of the duplication. In particular, how would you write this without an `if`?